PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+  - **Inactive**: The offer is inactive. It is not displayed on the marketplace and cannot be sold there.   
+  - **Offline**: The offer is unknown to the marketplace and therefore cannot be sold there.
+
+
+- [CANCEL]   
+  Click this button to cancel adding an offer and close the *Create Offer* window.
+
+- [SAVE]   
+  Click this button to save the offer and close the *Create Offer* window.
+
+
+### Add to multiple connections
+*PIM > Products > Tab LIST > Button Add > Button CREATE > Tab Offers > Button Add > Button Add to multiple connections*
+
+#### Select connections and status
+
+![Add to multiple connections](../../Assets/Screenshots/PIM/Products/List/Offers/MultipleConnections01.png "[Add to multiple connections]")
+
+**Select status and change tracking mode**
+
+- *Status*    
+  Click the drop-down list to select the connection status. The following statuses are available:    
+  - **Active**: The offer is active. It is displayed on the marketplace and can be sold there.
+  - **Inactive**: The offer is inactive. It is not displayed on the marketplace and cannot be sold there.   
+  - **Offline**: The offer is unknown to the marketplace and therefore cannot be sold there.
+
+
+- Select the appropriate change tracking mode. The following options are available:   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *manual*: Changes in a PIM product must be triggered manually to be applied to the offer. The initial offer is automatically applied.   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *semi-automatic*: Changes in a PIM product are automatically applied to the offer after confirmation in the *Omni-Channel* module. The initial offer is automatically applied without confirmation.   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *semi-automatic, changes must be confirmed by another user*: Changes in a PIM product are automatically applied to the offer after confirmation in the *Omni-Channel* module by another user. The initial offer is only applied after confirmation by another user.    
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *automatic*: Changes in a PIM product are automatically applied to the offer. The initial offer is also automatically applied.
+
+
+**Select connections**
+
+- [SELECT ALL]   
+  Click this button to select all available connections.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") Connection name     
+  Enable a toggle to select the corresponding connection to create an offer to. All available connections are displayed. By default, all toggles are disabled.
+
+- [CANCEL]   
+  Click this button to cancel adding multi-connection offers and close the *Select Connections and status* wizard window.
+
+- [CONTINUE]   
+  Click this button to proceed to the next step to add multi-connection offers. The *Select destination attribute sets* wizard window is displayed.
```

Comment:
```suggestion
  Click this button to proceed to the next step (to add multi-connection offers). The *Select destination attribute sets* wizard window is displayed.
```
Ich würde generell vorschlagen, dass wir nicht so spezifisch sind (also Text in Klammern würde ich weglassen). So können wir den Satz wieder verwenden. Gleiches gilt für BACK/NEXT... Für allgemeine Plattform-Funktionen/Buttons usw., die modulübegreifend vorkommen, würde ich so weit wie möglich standarisieren.